### ISSUE TITLE
No user creation in ldap with config.ldap_create_user = true in the Devise Initializer

### ISSUE THREAD

**Issue by blischalk** [cschiewek/devise_ldap_authenticatable#153]
On a clean Rails 4 install I am testing out Devise with devise_ldap_authenticatable.

With config.ldap_create_user = true set in the Devise initializer, when I signup a new user no entry is created in the ldap.  While tailing the ldap logs I see no request being made to create the user.  While observing the development log output in Rails it looks as if it is just querying the Rails relational database instead of calling out to the LDAP.

As well as the user not getting created, the user is "Logged In" anyway even though no ldap entry was created.  After logging out and attempting to login again with the credentials that the user used to signup, the user is unable to login.

It seems that if this config.ldap_create_user is set to true that if no entry is created in the ldap for whatever reason the user should not be logged in anyway.

Is there some other configuration that I may be missing?

**Comment by Startouf** [comment]
Even if this is off-topic, I've been looking for this too ! This should be added to the gem if someone's willing to code it.

**Comment by jozefvaclavik** [comment]
@Startouf  I think the main issue here is that different servers have different requirements. Eventually you would have to get down to the code and figure out what attributes you need to set up for your server.. If you ignore all attributes from the sample, it is 3 lines of code anyway. Seems already pretty simple..

**Comment by Startouf** [comment]
Yes you're right, my point was that I didn't know it was possible to add to the ldap using this gem. I thought I had to fall back to Net::Ldap to do this (though it's most likely not that much more difficult)

**Comment by stevenpy** [comment]
Any news ? Still looking for that
